app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -371,39 +320,15 @@ class NavigationDrawerTestActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
   fun testNavigationDrawerTestActivity_withUserProfile_openNavigationDrawer_checkAdministratorControlsNotDisplayed() { // ktlint-disable max-line-length
     launch<NavigationDrawerTestActivity>(
       createNavigationDrawerActivityIntent(
         internalProfileId1
       )
     ).use {
-      onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
+      it.openNavigationDrawer()
       onView(withId(R.id.administrator_controls_linear_layout))
-        .check(
-          matches(
-            not(
-              isDisplayed()
-            )
-          )
-        )
-    }
-  }
-
-  @Test
-  // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
-  fun testNavigationDrawerTestActivity_openNavigationDrawer_selectHelpMenuInNavigationDrawer_showsHelpFragmentSuccessfully() { // ktlint-disable max-line-length
```

Comment:
Just to check--was this removed because testNavigationDrawerTestActivity_openNavDrawer_selectHelpMenu_opensHelpActivity effectively tests this flow by verifying HelpActivity navigation?

Reply:
yes